src/WorkerChannel.ts

```diff
@@ -61,4 +69,12 @@ export class WorkerChannel {
             after(context);
         }
     }
+
+    public async updatePackageJson(dir: string): Promise<void> {
+        try {
+            this.packageJson = await readJson(path.join(dir, 'package.json'));
+        } catch {
+            this.packageJson = {};

```

Comment:
Let's warn users when this happens. Similar to [this](https://github.com/Azure/azure-functions-nodejs-worker/blob/v3.x/src/eventHandlers/functionLoadRequest.ts#L22), except a warning

Reply:
Done!